### PR TITLE
docs(trino): add note about specifying RBAC role with Starburst

### PR DESCRIPTION
**Pull request by gforsyth**

### CODE REVIEW

**Comment by gforsyth** [comment]
Don't merge this -- going to poke at starburst a little bit first

**Comment by gforsyth** [comment]
Ok, Starburst doesn't appear to require specifying a role in all situations, so I left a note in the docs to try it if there are issues listing tables.

Also added in some guidance in determining what to use for connecting to a starburst trino server.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -33,6 +33,7 @@ repos:
     rev: v2.2.6
     hooks:
       - id: codespell
+        entry: codespell --skip="*.js,*.html,*.css,*.svg"
```
Comment (cpcloud) [inline .pre-commit-config.yaml:36]: Should we add this to the pyproject.toml config?

**Review by cpcloud — APPROVED** [review]

File: .pre-commit-config.yaml (hunk 1 above)
Comment (gforsyth) [reply .pre-commit-config.yaml:36]: yeah, I'll add that

Comment (cpcloud) [reply .pre-commit-config.yaml:36]: In theory you should be able to only declare it in `pyproject.toml` and it'll get picked up by the pre-commit invocation

Comment (gforsyth) [reply .pre-commit-config.yaml]: yep, that worked.

**Pull request merged**